docs/docsite/rst/network/user_guide/platform_eos.rst

```diff
@@ -74,13 +65,13 @@ Example CLI ``group_vars/eos.yml``
 - If you are accessing your host directly (not through a bastion/jump host) you can remove the ``ansible_ssh_common_args`` configuration.
 - If you are accessing your host through a bastion/jump host, you cannot include your SSH password in the ``ProxyCommand`` directive. To prevent secrets from leaking out (for example in ``ps`` output), SSH does not support providing passwords via environment variables.
 
-Example CLI Task
+Example CLI task
 ----------------
 
 .. code-block:: yaml
 
    - name: Backup current switch config (eos)
-     eos_config:
+     arista.eos.eos_config:
        backup: yes
      register: backup_eos_location
      when: ansible_network_os == 'eos'
```

Comment:
With collection the value if network_os is `arisat.eos.eos` so this conditional will be `when: ansible_network_os == 'arista.eos.eos'